wiki/Difficulties/en.md

```diff
@@ -146,4 +148,4 @@ Rest moments in Taiko
 -   ![Hard](/wiki/shared/Hard-t.png "Hard") Main snapping 1/1 + 1/2 with 3/2 or 2/1 breaks
 -   ![Insane](/wiki/shared/Insane-t.png "Insane") Main snapping 1/2 + 1/4 with 1/1 breaks
 
-For skilled mappers / players who are having problems to distinguish when a lower difficulty becomes too hard, you can try playing your beatmap on **[T-Aiko!](T-Aiko! "wikilink")** which gives you a different difficulty feeling and can show how lower player could feel when playing your maps.
+For skilled mappers / players who are having problems to distinguish when a lower difficulty becomes too hard, you can try playing your beatmap on **[T-Aiko!](/wiki/Game_Modes/External_Ports/T-Aiko/Old)** which gives you a different difficulty feeling and can show how lower player could feel when playing your maps.
```

Comment:
L151: Seems unecessary since you can just request a playtest from players with different skillsets